src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TelemetryChannelEventSource.cs

```diff
@@ -417,6 +417,24 @@ public void LastBackendResponseWhenPutToStorage(int statusCode, double currentDe
             this.WriteEvent(58, statusCode, currentDelayInSeconds, this.ApplicationName);
         }
 
+        [Event(59, Message = "Error dequeuing file: {0}. Exception: {1}.", Level = EventLevel.Error)]
+        public void TransmissionStorageDequeueIOError(string fileName, string exception, string appDomainName = "Incorrect")
+        {
+            this.WriteEvent(59, fileName, exception, this.ApplicationName);
+        }
+
+        [Event(60, Message = "Unauthorized access dequeuing file: {0}. Exception: {1}.", Level = EventLevel.Warning)]
```

Comment:
Those two messages are warnings. Do you think either of them may be user-actionable? Maybe user can clean up the storages?